### PR TITLE
Problem: need to add rpc endpoint personal_unpair

### PR DESCRIPTION
**Pull request by Muggle-Du**
* this is aimed at smartcard wallet which is not supported yet.

* will return unsupported error now.

Closes: #730 

## Description

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: rpc/ethereum/namespaces/personal/api.go [hunk 1]
```diff
@@ -222,3 +222,11 @@ func (api *PrivateAccountAPI) EcRecover(_ context.Context, data, sig hexutil.Byt
 
 	return crypto.PubkeyToAddress(*pubkey), nil
 }
+
+// Unpair deletes a pairing between wallet and ethermint.
+func (api *PrivateAccountAPI) Unpair(_ context.Context, url string, pin string) error {
```
Comment (fedekunze) [inline rpc/ethereum/namespaces/personal/api.go:227]:
```suggestion
func (api *PrivateAccountAPI) Unpair(_ context.Context, url, pin string) error {
```

File: tests/rpc/personal_test.go [hunk 2]
```diff
@@ -142,3 +143,13 @@ func TestPersonal_LockAccount(t *testing.T) {
...
 }
+
+func TestPersonal_Unpair(t *testing.T) {
+	t.Skip("skipping TestPersonal_Unpair")
+
+	rpcRes := Call(t, "personal_unpair", []interface{}{"", 0})
+
+	var res error
+	err := json.Unmarshal(rpcRes.Result, &res)
+	require.Equal(t, fmt.Errorf("smartcard wallet not supported yet"), err)
```
Comment (fedekunze) [inline tests/rpc/personal_test.go:154]: use `errors.Is`

**Review by fedekunze — APPROVED** [review]
ACK. Minor comments, can you also add a changelog entry?

File: tests/rpc/personal_test.go (hunk 2 above)
Comment (Muggle-Du) [reply tests/rpc/personal_test.go:154]: thanks for your advice and correction.
I will add the changelog entry and make the correction.

Comment (Muggle-Du) [reply tests/rpc/personal_test.go]: changelog entry added and correct to use errors.Is

**Pull request merged**